R/real_vector.R

```diff
@@ -353,10 +353,18 @@ inspect_probability_vector <- function(
     return(res)
   }
 
+  value_all <- value
   value <- value[!is.na(value)]
 
   if (!(all(value >= 0) && all(value <= 1))) {
-    return(failure("Some values were outside the allowed range"))
+    mask <- value_all[value_all < 0 | value_all > 1]
+    return(failure(
+      paste0(
+        "Vector at position", ifelse(length(mask) > 1, "s ", " "),
+        paste(which(value_all %in% mask), sep = "' '", collapse = ", "),
```

Comment:
I am not sure the above predicate actually returns the appropriate indexes. It might, but it's pointless. All you need to print out are the indexes where the mask (the actual mask) is true.